ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -628,6 +628,20 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-ovn
+- as: aws-ipi-ovn-hypershift-public-private-mgmt-p1-f7
```

Comment:
@LiangquanLi930 could you please change `as: aws-ipi-ovn-hypershift-public-private-mgmt-p1-f7` to `as: aws-ipi-ovn-hypershift-mgmt-public-private-p1-f7` so that it align with other test with`hypershift-mgmt`?

Reply:
@kuiwang02  `public-private` is our special cluster， mgmt represents our testing angle，we will also have`aws-ipi-ovn-hypershift-public-private-guest-p1-f7` ,so I think it should be keep it

Reply:
your definition is <cluster env type>-<hypershift type>-<test angle>, and my suggestion is <cluster env type>-<test angle>-<hypershift type>
if exchanging it, what is the impact?

why I suggest it is to keep compatible.
I just fear some existing code which parse this tile are handling `hypershift-mgmt` or `hypershift-guest`, and your change will break it.

Reply:
We also designed it like this before, at least for now we have no influence, eg: `aws-ipi-ovn-hypershift-mce-mgmt-p2-f7`

Reply:
So I think it's easier to understand our pipeline, test type, hypershift type and ocp type

Reply:
ok. if you want to keep it, I have no comment.

Reply:
Suggest to sort it after `aws-ipi-ovn-hypershift-p2-f21`